help-docs/FAQ/collab-token.md

```diff
@@ -0,0 +1,51 @@
+# Get $COLLAB Tokens
+
+If you have recently created a new wallet address and are ready to acquire $COLLAB tokens, follow these steps to successfully add $COLLAB tokens to your new wallet:
+
+## Step 1: Add Optimism Network to Your Wallet

```

Comment:
this section is very long and makes the article lengthy. maybe let's only have:

> you can add the Optimism network to your MetaMask wallet by going to [https://chainlist.org/chain/10](https://chainlist.org/chain/10)

because 95% of people will be using MetaMask and if they are using any other wallet and they are a dev, they will know how to add it.